### PR TITLE
Drag and drop: fix misplaced drop indicator

### PR DESCRIPTION
**Pull request by ellatrix**
## What?

|Before|After|
|-|-|
|image|image|

## Why?

It's a bug.

## How?

We should ensure the width of the popover anchor does not become negative when the next and previous item are on a different line. 

Perhaps we should explore showing the inserter in between the line as a follow-up.

## Testing Instructions

Create a block with inner block showing horizontally and make the items wrap. Drag an item to the end of the first line.

### CODE REVIEW

File: packages/block-editor/src/components/block-popover/inbetween.js [hunk 1]
```diff
@@ -131,22 +131,21 @@ function BlockPopoverInbetween( {
...
+						width = nextRect.left - previousRect.right;
+						// Avoid a negative width when the next rect is on
+						// the next line.
+						width = Math.max( width, 0 );
+					}
+
 					if ( isRTL() ) {
 						// non vertical, rtl
 						left = nextRect ? nextRect.right : previousRect.left;
-						width =
```
Comment (youknowriad) [inline packages/block-editor/src/components/block-popover/inbetween.js]: I don't understand why you removed the isRTL case?

Comment (Mamaduka) [reply packages/block-editor/src/components/block-popover/inbetween.js]: We could move it inline on L135 - `isRTL() ? previousRect.left - nextRect.right : nextRect.left - previousRect.right;`

Comment (ellatrix) [reply packages/block-editor/src/components/block-popover/inbetween.js]: I misread it, I though it's the same and only `left` was different.

Comment (ellatrix) [reply packages/block-editor/src/components/block-popover/inbetween.js]: Fixed

**Review by youknowriad — APPROVED** [review]

**Review by ntsekouras — APPROVED** [review]
LGTM, thanks!

**Comment by ellatrix** [comment]
Thanks!

**Pull request merged**